### ISSUE TITLE
MacOS binary is freezing

### ISSUE THREAD

**Issue by prsolucoes** [bblanchon/pdfium-binaries#26]
Hi,

I made a iOS/MacOS repository that build it with success. Butt when execute my app with my static library for macOS it is freezing after open document:
https://travis-ci.com/prsolucoes/mobile-pdfium/jobs/279906862

My repository:
https://github.com/prsolucoes/mobile-pdfium

Do you know what can be wrong?

**Comment by jpap** [comment]
I also noticed this problem, and I have built pdfium from scratch.  I have tried the latest master branch and two older tags.  The included tests all pass without problem.

When it hangs, I see the program is chewing up a lot of CPU when I call `FPDF_LoadDocument`.

I wonder [if it is related to this bug](https://bugs.chromium.org/p/pdfium/issues/detail?id=1460)?  Unfortunately I (yet) can't seem to get debug symbols working so that I can try stepping through the code under Xcode, which would help track down the problem.
